module/Swissbib/src/Swissbib/Util/Config/ValueConverter.php

```diff
@@ -0,0 +1,221 @@
+<?php
```

Comment:
Why do we need this class?

Reply:
I've introduced this class because all values read from a config file are strings and at some point between reading the file and using the values in the client-side JavaScript I have to convert strings into whatever they are expected to be actually. So I decided to do it as early as possible. The component allows for automatic typed value conversion of several common formats. I thought it is worth the 5 extra minutes to make it handle the most typical stuff.

But its much like for FlatArrayConverter: I can remove the private method comments to reduce the noise.